Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM node:6-slim
+FROM node:10.20.1-slim
 
 WORKDIR /usr/src/app
 
 COPY package.json /usr/src/app
 
 RUN apt-get update \
-    && apt-get install -y jq --no-install-recommends \
+    && apt-get install -y jq python git build-essential --no-install-recommends \
```

Comment:
Do you really need `python` and `build-essential` at *runtime*, or only at 'build' time (and could they as such be removed within this very same `RUN` statement)?

Reply:
`python` and `build-essential` are only required at `build` time, and, yes, the can be removed.